zanata-war/src/main/resources/db/changelogs/db.changelog-3.8.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+
+  <changeSet id="1" author="aeng@redhat.com">
+    <comment>WebhookKey to HProject</comment>
+    <addColumn tableName="HProject">
+      <column name="webhookKey" type="varchar(255)">
```

Comment:
I think webhookSecret would be a bit clearer.  It would be good to use that term (or at least "secret") consistently in the code.